kolibri/plugins/device/assets/src/views/ManageContentPage/SelectNetworkAddressModal/SelectAddressForm.vue

```diff
@@ -9,68 +9,90 @@
         appearance="basic-link"
-        :text="$tr('refreshAddressesButtonLabel')"
-        @click="refreshSavedAddressList"
+        @click="$emit('click_add_address')"
       />
-    </UiAlert>
 
-    <KButton
-      v-show="!newAddressButtonDisabled"
-      class="new-address-button"
-      :text="$tr('newAddressButtonLabel')"
-      appearance="basic-link"
-      @click="$emit('click_add_address')"
-    />
+      <template v-for="(a, idx) in savedAddresses">
+        <div :key="`div-${idx}`">
+          <KRadioButton
+            :key="idx"
+            v-model="selectedAddressId"
+            class="radio-button"
+            :value="a.id"
+            :label="a.device_name"
+            :description="a.base_url"
+            :disabled="!a.available || !a.hasContent"
+          />
+          <KButton
+            :key="`forget-${idx}`"
+            :text="$tr('forgetAddressButtonLabel')"
+            appearance="basic-link"
+            @click="removeSavedAddress(a.id)"
+          />
+        </div>
+      </template>
 
-    <template v-for="(a, idx) in savedAddresses">
-      <div :key="`div-${idx}`">
-        <KRadioButton
-          :key="idx"
-          v-model="selectedAddressId"
-          class="radio-button"
-          :value="a.id"
-          :label="a.device_name"
-          :description="a.base_url"
-          :disabled="!a.available || !a.hasContent"
-        />
-        <KButton
-          :key="`forget-${idx}`"
-          :text="$tr('forgetAddressButtonLabel')"
-          appearance="basic-link"
-          @click="removeSavedAddress(a.id)"
-        />
-      </div>
-    </template>
+      <hr v-if="discoveredAddresses.length > 0">
 
-    <hr v-if="discoveredAddresses.length > 0">
-
-    <template v-for="d in discoveredAddresses">
-      <div :key="`div-${d.id}`">
-        <KRadioButton
-          :key="d.id"
-          v-model="selectedDeviceId"
-          class="radio-button"
-          :value="d.id"
-          :label="$tr('peerDeviceName', {identifier: d.id})"
-          :description="d.base_url"
-          :disabled="d.disabled"
-        />
-      </div>
+      <template v-for="d in discoveredAddresses">
+        <div :key="`div-${d.id}`">
+          <KRadioButton
+            :key="d.id"
+            v-model="selectedAddressId"
+            class="radio-button"
+            :value="d.instance_id"
+            :label="$tr('peerDeviceName', {identifier: d.id.slice(0,4) })"
+            :description="d.base_url"
+            :disabled="!d.available || !d.hasContent"
+          />
+        </div>
+      </template>
     </template>
 
-    <template v-if="false">
-      {{ $tr('searchingText') }}
-    </template>
+    <KFixedGrid slot="actions" class="actions" numCols="4">
+      <KFixedGridItem span="1">
+        <transition name="fade">
+          <div v-if="discoveringPeers" class="searching-indicator">
+            <KCircularLoader :size="16" :stroke="9" />{{ $tr('searchingText') }}
```

Comment:
We have a `KLabeledIcon` component for consistent spacing in situations like this. Also lowered the stroke width a bit.

```suggestion
            <KLabeledIcon>
              <KCircularLoader slot="icon" :size="16" :stroke="6" />
              {{ $tr('searchingText') }}
            </KLabeledIcon>
```

| before | after |
|--|--|
| ![image](https://user-images.githubusercontent.com/2367265/70168500-da744f00-167d-11ea-8341-b3eaa67e6c64.png) | ![image](https://user-images.githubusercontent.com/2367265/70168471-ca5c6f80-167d-11ea-88e2-32fda95489a3.png)|